Sources/DeepLearning/Layer.swift

```diff
@@ -1404,12 +1409,75 @@ public struct LSTMCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNumeric
     }
 }
 
+/// A gated recurrent unit (GRU) cell.
+///
+/// Reference: [Gated recurrent unit](https://en.wikipedia.org/wiki/Gated_recurrent_unit).
+@_fixed_layout
+public struct GRUCell<Scalar: TensorFlowFloatingPoint>: RNNCell {
+    public var resetWeight, updateWeight, inputHiddenWeight, hiddenHiddenWeight: Tensor<Scalar>
+    public var resetBias, updateBias, inputHiddenBias, hiddenHiddenBias: Tensor<Scalar>
```

Comment:
```suggestion
    public var b_r, b_u, b_ih, b_hh: Tensor<Scalar>
```
and similarly, in the rest of the code, what do you think?